projects/components/src/lib/infinite-scroll/infinite-scroll.component.ts

```diff
@@ -0,0 +1,31 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SpinnerComponent } from '../spinner/spinner.component';
+
+@Component({
+    selector: 'mz-infinite-scroll',
+    standalone: true,
+    imports: [CommonModule, SpinnerComponent],
+    templateUrl: './infinite-scroll.component.html',
+    styleUrl: './infinite-scroll.component.scss',
+})
+export class InfiniteScrollComponent {
+    @Input() items: any[] = [];
```

Comment:
It will be likely that we want to customize item display, which means....we can have `items`, `containerHeight`, `itemSelected` to be projected content or something to allow that customization.

Reply:
this is enabled now